src/libraries/System.Security.Permissions/src/System/Security/Policy/PolicyLevel.cs

```diff
@@ -8,43 +8,43 @@ namespace System.Security.Policy
     public sealed partial class PolicyLevel
     {
         internal PolicyLevel() { }
-        [Obsolete("Because all GAC assemblies always get full trust, the full trust list is no longer meaningful. You should install any assemblies that are used in security policy in the GAC to ensure they are trusted.")]
+        [Obsolete]
```

Comment:
Those messages never reached a consumer as the reference assembly doesn't have them. You're probably right that those should also be on the reference assembly instead of deleting them in the source.

Reply:
Added them to the ref.